client/src/components/AuditAdmin/Progress/Progress.tsx

```diff
@@ -501,6 +524,17 @@ const Progress: React.FC<IProgressProps> = ({
           auditSettings={auditSettings}
         />
       )}
+      {jurisdictionDiscrepanciesId && round && (
```

Comment:
Why is `round` needed in this check?

Reply:
Hm slipped in, not needed as we already hide discrepancies and thus the ability to select one if `round` is not set [here](https://github.com/votingworks/arlo/pull/2033/files#diff-e6c71ff26929b295d562cf958683bb2cd9259f1810dd15ee35cd7bf8b0222363R80). Removing